worker/introspection/pprof/pprof.go

```diff
@@ -163,6 +166,44 @@ func Symbol(w http.ResponseWriter, r *http.Request) {
 	w.Write(buf.Bytes())
 }
 
+func durationExceedsWriteTimeout(r *http.Request, seconds float64) bool {
+	srv, ok := r.Context().Value(http.ServerContextKey).(*http.Server)
+	return ok && srv.WriteTimeout != 0 && seconds >= srv.WriteTimeout.Seconds()
```

Comment:
technically, exceeds means > rather than >=

Reply:
Sure :D but as a direct copy from go 1.9, I did not want to introduce too much difference.